os/pm/pm_changestate.c

```diff
@@ -110,6 +110,10 @@ static void pm_timer(int domain)
 
 	if (!pdom->wdog) {
 		pdom->wdog = wd_create();
+		if (!pdom->wdog) {
+			pmdbg("Fail to create wdog\n");
```

Comment:
Since this is error scenario, should we use error log instead of dbg log?

Reply:
Thank you for review
It is calling by idle thread. so It is appred to change to the lldbg.

I have chagned for svace, but we are not using this function. so let me clean-up the this code.
thank you~